parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -90,7 +87,7 @@ public void visit(ThriftType.MapType mapType) {
     //=========handle value
     currentFieldPath.push(mapValueField);
     currentName = "value";
-    currentRepetition = OPTIONAL;
+    currentRepetition = OPTIONAL; // TODO: why?
```

Comment:
I should double check if we enforce that you can't project fields nested under a key